src/alpaca_typer.erl

```diff
@@ -2054,7 +2053,7 @@ typ_apply(Env, Lvl, TypF, NextVar, Args, Line) ->
                     {error, _}=Err -> Err;
                     {Typ, NV} ->
                         case get_cell(Typ) of
-                            {t_receiver, Recv2, RetTyp} ->
+                            {t_receiver, _Recv2, RetTyp} ->
```

Comment:
This looks a bit odd, somehow. Why is this called `Recv2` and then ignored? Is that really intended?